.storybook/main.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
     ];
 
     // remove default storybook svg loader
-    config.module.rules = config.module.rules.map((rule) => {
+    config.module.rules = config.module.rules.map(rule => {
```

Comment:
is this change effect of using proper prettier config or the previous one? :)

Reply:
Yep. Since we dont have project config for prettier I'm using same version as our eslint config.( from `/node_modules/eslint-config-brainly/prettier.config.js`)